app/uk/gov/hmrc/eusubsidycompliancefrontend/persistence/UndertakingCache.scala

```diff
@@ -118,7 +118,7 @@ class UndertakingCache @Inject() (
 }
 
 object UndertakingCache {
-  val DefaultCacheTtl: FiniteDuration = 24 hours
+  val DefaultCacheTtl: FiniteDuration = 1 hours
```

Comment:
`replaceIndexes` parameter in MongoCacheRepository should be set to true, otherwise it might fail or not work as expected.